package.json

```diff
@@ -26,6 +26,7 @@
     "build:post-tsc": "yarn workspaces foreach --parallel --topological --verbose run build:post-tsc",
     "clean": "yarn workspaces foreach --parallel --verbose run clean",
     "test": "yarn workspaces foreach --parallel --verbose run test",
+    "test:browser": "yarn workspaces foreach --verbose run test:browser",
```

Comment:
why `test` doesn't run `test:browser` as well

Reply:
It does. I used this command to only run browser tests, but it's obsolete now.